js/gestionPresupuesto.js

```diff
@@ -1,18 +1,44 @@
-// TODO: Crear las funciones, objetos y variables indicadas en el enunciado
+let presupuesto = 0;
 
-// TODO: Variable global
-
-
-function actualizarPresupuesto() {
-    // TODO
+function actualizarPresupuesto(cantidad) {
+    if (typeof cantidad === 'number' && cantidad >= 0) {
+        presupuesto = cantidad;
+        return presupuesto;
+    } else {
+        console.log("La cantidad introducida no puede ser negativa.")
+        return -1;    
+    }
 }
 
 function mostrarPresupuesto() {
-    // TODO
+    return "Tu presupuesto actual es de " + presupuesto + " €";
 }
 
-function CrearGasto() {
-    // TODO
+function CrearGasto(descripcion, cantidad) {
+
+    let valor = 0;
+
+    if (typeof cantidad === 'number' && cantidad >= 0) {
+        valor = cantidad;
+    }
+
+    let gasto = {
```

Comment:
No debes crear un objeto literal. En lugar de eso debes establecer las propiedades mediante `this`, ya que se trata de una función constructora: https://es.javascript.info/constructor-new#funcion-constructora

Corrígelo y vuelve a abrir la PR cuando lo tengas.

Un saludo!!

Reply:
Por lo de los commits, no te preocupes :wink: